### ISSUE TITLE
Custom Folder Summary w/Images View

### ISSUE THREAD

**Issue by paulrentschler** [PSUEducationalEquity/plonetheme.eeq#45]
As a user I can see the contents of a folder displayed in "Summary View" using title and text styles consistent with the rest of the site and the lead image displayed to the left of the title and description. Each entry should have a drop shadow to separate it from the others.

This view will be chosen by content editors when it is known that all content items have a lead image.

We are using this for inspiration:
![image](https://user-images.githubusercontent.com/1778602/122450367-ed572780-cf74-11eb-9f9b-ea4c72d8cd57.png)

Which is a screenshot from: https://agsci.psu.edu/academics/undergraduate/majors/animal-science which uses Plone 5 and Mosaic.

**Comment by skleinfeldt** [comment]
@paulrentschler Can you please specify the image crop that should be used for this view? Remember about plone.app.imagecropping and your ability to define specific crops #25. We've defined Hero, Tile, and Square crops so far, hopefully one of those will do?

**Comment by skleinfeldt** [comment]
Also, we should probably call this "Summary View with Images" to be clear, yes?

**Comment by MBrahosky** [comment]
@skleinfeldt the image crop could be the same as the square image already defined. The 220x220 image crop.

**Comment by paulrentschler** [comment]
And yes "Summary View with Images" is perfect

**Comment by silviot** [comment]
@paulrentschler I added a new `Summary View with Images` view. To see it you'll need to pull in the latest code and uninstall/reinstall the `plonetheme.eeq` product on https://beta.equity.psu.edu/web/prefs_install_products_form

**Comment by paulrentschler** [comment]
[Example page we are looking at](https://beta.equity.psu.edu/web/about/awards)

## Tweaks:

* remove the `Read More` section
* the object title is an H3 in the AgSci site, not sure if that makes sense for us but that text should appear slightly larger than the summary text and bold.
* the summary text should not be underlined on hover (seeing this in Firefox not Chrome)
* in Chrome nothing gets underlined and the title should be
* this view appears to only be available for Folders, can it also be made available for Collections?

## Logic for what date to display

* if it is an Event object, display the start date if one is set
* if the object has an effective date set, display it
* do not display anything for a date

**Comment by silviot** [comment]
@paulrentschler [Done](https://github.com/PSUEducationalEquity/plonetheme.eeq/commit/729880d098b2a13163dadc96e490ba5bcb14eaaa).

**Comment by paulrentschler** [comment]
@silviot these changes all look great.

Is it possible to get this "Folder Summary with Images" view available on collections as well?

**Comment by silviot** [comment]
@paulrentschler I think I understand: that menu item I was referring to controls the view of the container of the collection.

**Comment by paulrentschler** [comment]
@silviot Yes, I think you have it now. If the folder has a collection as its default view like [this page](https://beta.equity.psu.edu/web/offices/student-disability-resources/events) the display menu looks like this: 
![image](https://user-images.githubusercontent.com/1778602/150563781-0dd458a8-98e1-4eb9-b67f-560b4553e72e.png)

**Comment by silviot** [comment]
@paulrentschler I'm surprised it does not show up: it did for me locally when I added [this ZCML directive](https://github.com/PSUEducationalEquity/plonetheme.eeq/blob/master/src/plonetheme/eeq/browser/configure.zcml#L86-L94).
I [added a directive to the profile](https://github.com/PSUEducationalEquity/plonetheme.eeq/commit/9ddd1e5497bff24890e8036dc5055956c895861c) to make it the same as the (working) additional folder view. Can you retry?

**Comment by paulrentschler** [comment]
@silviot I did the usual git pull, buildout, restart with no success. Then I uninstalled and reinstalled the plonetheme.eeq add on and that resolved the issue. The view is now available for collections.
